core/controllers/base.py

```diff
@@ -923,8 +904,13 @@ def _create_token(cls, user_id: Optional[str], issued_on: float) -> str:
         # Round time to seconds.
         issued_on_str = str(int(issued_on))
 
+        csrf_secret_model = auth_services.get_csrf_secret_model()
```

Comment:
Don't expose the model all the way up in the controller. Have auth_services() return the value directly, instead.

Reply:
Updated